src/test/java/com/titarenko/rest/HttpControllerTest_withoutMocks.java

```diff
@@ -12,68 +13,207 @@
 import java.net.http.HttpClient;
 import java.net.http.HttpRequest;
 import java.net.http.HttpResponse;
-import java.util.Collections;
-import java.util.List;
+import java.time.LocalDate;
+import java.util.*;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.*;
 
 public class HttpControllerTest_withoutMocks {
 
     private static final HttpClient HTTP_CLIENT = HttpClient.newHttpClient();
     public static final JsonParser JSON_PARSER = new JsonParser();
-    private static Employee vasil = new Employee.Builder().withName("Vasil").withSalary(150).build();
+    private static List<Employee> database;
+    private static Employee vasil = new Employee.Builder()
+            .withId(1)
+            .withName("Vasil")
+            .withSex(Gender.M)
+            .withPosition("position")
+            .withSalary(150)
+            .withDateOfHire(LocalDate.parse("2012-12-12"))
+            .build();
+    private static Employee petr = new Employee.Builder()
+            .withId(2)
+            .withName("Petr")
+            .withSex(Gender.M)
+            .withPosition("position2")
+            .withSalary(150)
+            .withDateOfHire(LocalDate.parse("2011-11-11"))
+            .build();
+    private static Employee stepa = new Employee.Builder()
+            .withId(3)
+            .withName("Stepa")
+            .withSex(Gender.M)
+            .withPosition("position3")
+            .withSalary(200)
+            .withDateOfHire(LocalDate.parse("2010-10-10"))
+            .build();
+
+    private String incomingJson = """
+                {
+                  "id" : 10,
+                  "name" : "Tom Cruz",
+                  "sex" : "M",
+                  "position" : "actor",
+                  "salary" : 350,
+                  "dateOfHire" : "2010-10-10"
+                }
+                """;
 
     @BeforeAll
     static void startHttpController() {
+        database = new ArrayList<>();
```

Comment:
Run this initialization **_@BeforeEach_** test. Also, add:
```
    @AfterEach
    public void closeHttpController() {
        httpController.close();
    }
```
For this you have to add to `class Controller` a method wich stops a HttpServer:
```
public class Controller {
    private HttpServer server;
    public Controller(EmployeeService service) {
        this.service = service;
        try {
            server = HttpServer.create(new InetSocketAddress(1408), 0);
            .......
    }

    public void close(){
        server.stop(0);
    }
```